autoscheduler/frontend/src/package.json

```diff
@@ -7,7 +7,7 @@
     "test": "jest",
     "dev": "webpack --mode development ./index.tsx --output ../static/main.js",
     "build": "webpack --mode production ./index.tsx --output ../static/main.js",
-    "lint": "eslint ."
+    "lint": "eslint --ext .js,.jsx,.ts,.tsx ."
```

Comment:
While it definitely wasn't working for me previously, even with this new change I get no output when I run `npm run lint`. Any ideas?

Reply:
Try intentionally breaking a file, then running it again. Mine succeeds silently and only produces output if there's a problem

Reply:
Yeah this ended up working, all good